src/absil/ilwrite.fs

```diff
@@ -1401,7 +1401,7 @@ and GenCustomAttrPass3Or4 cenv hca attr =
     AddUnsharedRow cenv TableNames.CustomAttribute (GetCustomAttrRow cenv hca attr) |> ignore
 
 and GenCustomAttrsPass3Or4 cenv hca (attrs: ILAttributes) = 
-    attrs.AsList |> List.iter (GenCustomAttrPass3Or4 cenv hca) 
+    attrs.AsArray |> Array.iter (GenCustomAttrPass3Or4 cenv hca) 
```

Comment:
Why do we even need to do asArray? Just use for loop or Seq.iter directly on the attrs

Reply:
this is how ILAttributes is declared
```
[<NoEquality; NoComparison; Struct>]
type ILAttributes(array : ILAttribute[]) =

    member x.AsArray = array

    member x.AsList = x.AsArray |> Array.toList
```